app/Models/Customer.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.TooMany</br>**Severity**: error</br>**File**: app/Models/Customer.php</br>**Line**: 15</br>Expected 1 blank line at end of file; 3 found